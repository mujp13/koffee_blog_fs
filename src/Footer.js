import React from 'react';

export default function Footer() {
  return (
    <footer>
      2020 <a href="/">Koffee Blog</a> All rights reserved.
    </footer>
  );
}
