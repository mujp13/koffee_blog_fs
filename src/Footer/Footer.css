.footer {
  text-align: center;
}

img {
  align-items: center;
  width: 100%;
}
