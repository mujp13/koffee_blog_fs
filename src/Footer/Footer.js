import React from 'react';
import './Footer.css';

export default function Footer() {
  return (
    <>
      <footer className="footer">
        Welcome to <a href="/">Koffee Blog</a>
      </footer>
    </>
  );
}
