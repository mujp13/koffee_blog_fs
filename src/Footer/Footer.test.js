import React from 'react';
import ReactDOM from 'react-dom';
import Footer from './Footer';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <MemoryRouter>
      <Footer />
    </MemoryRouter>,
    div
  );

  ReactDOM.unmountComponentAtNode(div);
});

//snapshot
it('renders correctly', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
