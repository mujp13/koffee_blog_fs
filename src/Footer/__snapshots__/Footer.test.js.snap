// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="footer"
>
  Welcome to 
  <a
    href="/"
  >
    Koffee Blog
  </a>
</footer>
`;
